paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -180,7 +180,8 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
             customerConfig,
             paymentMethodTypes
         ).filter { paymentMethod ->
-            paymentMethod.hasExpectedDetails()
+            paymentMethod.hasExpectedDetails() &&
```

Comment:
Does this method return a list of saved on-session PMs for the customer? Is that why we are filtering out PayPal here? 

Can we add a comment here explaining why we are filtering out PayPal

Reply:
Updated.